css/properties/text-decoration-line.json

```
@@ -136,12 +136,12 @@
             ],
             "support": {
               "chrome": {
-                "version_added": "≤83"
+                "version_added": "57"
               },
               "chrome_android": "mirror",
               "edge": "mirror",
               "firefox": {
-                "version_added": "≤72"
+                "version_added": "≤10"
```

Tested [this example](http://bcd-examples.gh.caugner.de/css/text-decoration-line/) in Firefox 5 and 6 to establish that this was added in Firefox 6.

```suggestion
                "version_added": "6"
```